toolchains/xslt-M4/document/json/object-reference-html.xsl

```diff
@@ -58,15 +58,24 @@
       <xsl:variable name="header-tag" select="if ($level le 6) then ('h' || $level) else 'p'"/>
       <xsl:variable name="grouped-object" select="(self::array | self::singleton-or-array)/*"/>
       <xsl:variable name="me" select="($grouped-object,.)[1]"/>
+
       <div class="model-entry definition { tokenize($me/@_metaschema-json-id,'/')[2] }">
          <xsl:variable name="header-class" expand-text="true">{ if (exists(parent::map)) then 'definition' else 'instance' }-header</xsl:variable>
          <div class="{ $header-class }">
-            <!-- generates h1-hx headers picked up by Hugo toc -->
-            <xsl:element expand-text="true" name="{ $header-tag }" namespace="http://www.w3.org/1999/xhtml">
-               <xsl:attribute name="id" select="@_tree-json-id"/>
-               <xsl:attribute name="class">toc{ $level} name</xsl:attribute>
-               <xsl:text>{ @key }</xsl:text>
+            <!-- ===!!! generates h1-hx headers picked up by Hugo toc !!!=== -->
```

Comment:
Regarding my previous review recommendation, perhaps we can make the comment more precise like so?

```suggestion
            <!-- Generate the proper class attribute to match a given HTML header from $header-tag (h1, h2, h3) to define the proper style (toc1, toc2, toc3) as bound by toc{ $level }. This build-time generation of anchor from template improves performance of website at load time to decrease time to responsiveness. -->
```